lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -428,65 +428,83 @@ def _call(self, method: Callable, json_data: Dict[AnyStr, Any]):
         Returns:
             Postprocessed JSON body to return to client
         """
+        klasname = self.__class__.__name__
         try:
             es_request = self.assemble(json_data)
             path = es_request.get("path")
             url = urljoin(self.es_url, path)
         except Exception as e:
-            self.logger.exception("Blew it in setup: {}", type(e).__name__)
+            self.logger.exception("{} blew it in setup: {}", klasname, type(e).__name__)
```

Comment:
Just out of curiosity, why do you like `type(e).__name__` better than `str(e)`?

Reply:
Mostly, in this case, to give a concise summary without being overly redundant, since the `logger.exception` will dump everything. Sometimes it's hard to identify the actual exception from the stringification. Said stringification will appear in the exception dump anyway, so I just added the exception name.